app/services/flickr_service.rb

```diff
@@ -0,0 +1,11 @@
+class FlickrService
+  class << self
+    def search_photos(text, license, extras, per_page, page)
+      FlickRaw.api_key = ENV['FLICKR_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.